### DISCUSSION TITLE
Https-sniffing doesn't work for Android APK

### DISCUSSION THREAD

**Discussion by sergorl** [mitmproxy/mitmproxy#6508] [Intercept Everything!]
#### Problem Description
* I tried many versions of `mitmweb` (`9.0 - 10.0`) on `Windows 10` from official site as GUI web proxy for https-sniffing
* I unpinned my `apk` from origin certification by `apk-mitm`
* I installed proxy-certificate from `mitm.it` for `Anfroid apk`
* But https-sniffing happened only for `Android Chrome Browser`, not for `apk`
* Also I constantly see the next string in log:
```
Client TLS handshake failed. The client does not trust the proxy's certificate for ... (OpenSSL Error([('SSL routines', '', 'sslv3 alert certificate unknown')]))
```

#### Steps to reproduce the behavior:
1. `mitmweb -v --ssl-insecure`
2. unpinned `apk`
3. install `apk`
4. install mitmproxy certificate from `mitm.it` onto `Android phone`

#### System Information
```
Mitmproxy: 10.1.1 binary
Python:    3.11.5
OpenSSL:   OpenSSL 3.1.3 19 Sep 2023
Platform:  Windows-10-10.0.19045-SP0
```

**Comment by mhils** [comment]
https://docs.mitmproxy.org/stable/concepts-certificates/#certificate-pinning

**Reply by sergorl** [reply]
it doesn't help me unfortunately, I did previosly and there is no positive result
